compiler/sem.nim

```diff
@@ -526,7 +526,9 @@ proc semMacroExpr(c: PContext, n, nOrig: PNode, sym: PSym,
   if efNoSemCheck notin flags:
     result = semAfterMacroCall(c, n, result, sym, flags, expectedType)
   if c.config.macrosToExpand.hasKey(sym.name.s):
-    message(c.config, nOrig.info, hintExpandMacro, renderTree(result))
+    message(c.config, nOrig.info, hintExpandMacro, renderTree(result, {
+      renderNonExportedFields
+    }))
```

Comment:
Could also copy the flags from here, for the output of `repr(NimNode)`:

https://github.com/nim-lang/Nim/blob/9bb7e53e7f56fe3ddaa69849ff663ad18858e038/compiler/vm.nim#L1678

Reply:
Done in 1a73868e9d8b929393173b486d76198249fbfd1a